importlab/parsepy.py

```diff
@@ -38,18 +38,18 @@ def __new__(cls, name, new_name=None, is_from=False, everything=False,
           is_from: If the last part of the name (the "z" in "x.y.z") can
           be an element within a module, instead of a module itself. Happens
           e.g. for "from sys import argv".
-          everything: If this is an import of the form "from x import *".
+          is_star: If this is an import of the form "from x import *".
```

Comment:
I just noticed that 'source' is missing from this docstring: could you add it?

Reply:
done